src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -60,7 +61,9 @@ public ReadOnlyPerson getPerson() {
     public CommandResult execute() {
         try {
             addressBook.addPerson(toAdd);
-            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+            List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+            //return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
```

Comment:
Please remove unwanted code lines instead of leaving them as commented out chunks.